intel-sgx/ias/examples/attestation-tool.rs

```diff
@@ -0,0 +1,204 @@
+/* Copyright (c) Fortanix, Inc.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#[macro_use]
+extern crate clap;
```

Comment:
Since we are using Rust Edition 2018, we may just go with
```suggestion
use clap::clap_app;
```